blockly-core/blocks/functional_examples.js

```diff
@@ -31,7 +31,9 @@ goog.require('Blockly.Blocks');
  * Definition block for a custom functional block
  */
 Blockly.Blocks.functional_example = {
-  hideInMainBlockSpace: true,
+  hideInMainBlockSpace: function () {
```

Comment:
Maybe rename to `shouldHideInMainBlockSpace` now that it's a function just to clarify it's not a command